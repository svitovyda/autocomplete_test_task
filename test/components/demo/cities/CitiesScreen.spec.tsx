import { render } from '@testing-library/react';
import 'jest-styled-components';
import * as React from 'react';
import { CitiesScreen } from '../../../../src/components/demo/cities/CitiesScreen';

describe('CitiesScreen', () => {
  it('renders without error', () => {
    const component = render(<CitiesScreen />);

    expect(component).toMatchSnapshot();
  });
});
