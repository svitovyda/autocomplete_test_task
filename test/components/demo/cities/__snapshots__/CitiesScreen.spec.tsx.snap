// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesScreen renders without error 1`] = `
{
  "asFragment": [Function],
  "baseElement": .c1 {
  margin: 8px 0;
  display: block;
  width: 100%;
}

.c0 {
  max-width: 600px;
  margin: 0 auto;
  padding: 0 10px;
  display: block;
}

.c2 {
  width: 100%;
  margin: 0;
  padding: 0;
  display: block;
  height: 0;
}

<body>
    <div>
      <div
        class="c0"
      >
        <h1>
          Static Data: Cities
        </h1>
        <div
          class="c1"
        >
          <input
            placeholder="Start typing to find cities..."
            type="text"
            value=""
          />
          <div
            class="c2"
          />
        </div>
        <div>
          MapView Mock
          <pre>
            {"marker":{"id":"4","name":"Berlin","lat":52.52,"lng":13.405}}
          </pre>
        </div>
      </div>
    </div>
  </body>,
  "container": .c1 {
  margin: 8px 0;
  display: block;
  width: 100%;
}

.c0 {
  max-width: 600px;
  margin: 0 auto;
  padding: 0 10px;
  display: block;
}

.c2 {
  width: 100%;
  margin: 0;
  padding: 0;
  display: block;
  height: 0;
}

<div>
    <div
      class="c0"
    >
      <h1>
        Static Data: Cities
      </h1>
      <div
        class="c1"
      >
        <input
          placeholder="Start typing to find cities..."
          type="text"
          value=""
        />
        <div
          class="c2"
        />
      </div>
      <div>
        MapView Mock
        <pre>
          {"marker":{"id":"4","name":"Berlin","lat":52.52,"lng":13.405}}
        </pre>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
